### PR TITLE
Banner for 1.4.0 beta 

### PR DESCRIPTION
**Pull request by eed3si9n**
```
[info] welcome to sbt 1.4.0-SNAPSHOT (AdoptOpenJDK Java 1.8.0_232)
[info] loading settings for project global-plugins from ...
[info] loading global plugins from ...
[info] loading project definition from /private/tmp/hello/project
[info] loading settings for project root from build.sbt ...
[info] set current project to hello (in build file:/private/tmp/hello/)
[info]
[info] Here are some highlights of this release:
[info]   - Build server protocol (BSP) support
[info]   - sbtn: a thin client for sbt built as GraalVM native image
[info]   - VirtualFile + RemoteCache: caches build artifacts across different machines
[info]   - Incremental build pipelining. Try it using `ThisBuild / usePipelining := true`.
[info] See http://eed3si9n.com/sbt-1.4.0-beta for full release notes.
[info] Hide the banner for this release by running `skipBanner`.
[info] sbt server started at local:///Users/eed3si9n/.sbt/1.0/server/478e6db75688771ddcf1/soc
```

### CODE REVIEW

**Comment by eatkins** [comment]
I'm not sure I want to give oracle free advertising with announcing that it uses graalvm. Can we just say native thin client?

**Comment by eed3si9n** [comment]
I feel like the fact that it uses GraalVM would be a cool thing for `sbtn`, but it's up to you of course.

**Comment by eatkins** [comment]
To me it's first and foremost an implementation detail. I'm not sure I would use GraalVM if it didn't happen to make code sharing of the json codecs easier. GraalVM is not at all enjoyable to use because it's difficult to configure, the build times are extremely slow and it doesn't work with a lot of java libraries. It's a cool technology but I'm more excited that it shows what is possible wrt startup time and memory utilization while still being able to use an expressive language.

**Comment by eatkins** [comment]
Honestly, it might have been less work to write in c since I ended up having to implement ipcsocket anyway.

File: main/src/main/scala/sbt/internal/Banner.scala [hunk 1]
```diff
@@ -10,6 +10,16 @@ package sbt.internal
 private[sbt] object Banner {
   def apply(version: String): Option[String] =
     version match {
+      case v if v.startsWith("1.4.0") =>
+        Some(s"""
+                |Here are some highlights of this release:
+                |  - Build server protocol (BSP) support
+                |  - sbtn: a thin client for sbt built as GraalVM native image
```
Comment (eed3si9n) [inline main/src/main/scala/sbt/internal/Banner.scala]: so

```suggestion
                |  - sbtn: a native thin client for sbt
```

?

Comment (eatkins) [reply main/src/main/scala/sbt/internal/Banner.scala]: 👍

**Comment by eed3si9n** [comment]
`dependency-management/snapshot-local` is failing but likely it's unrelated, so I'm gonna merge this.

**Pull request merged**